src/main/java/eu/bbmri_eric/negotiator/database/model/Negotiation.java

```diff
@@ -146,13 +140,16 @@ public void setCurrentState(NegotiationState negotiationState) {
   public void setStateForResource(String resourceId, NegotiationResourceState state) {
     currentStatePerResource.put(resourceId, state);
     if (!state.equals(NegotiationResourceState.SUBMITTED)) {
+      System.out.println("here");
```

Comment:
is there a reason to have the println in here?

Reply:
sorry leftovers from debugging